### PR TITLE
evaluation-proxy documentation

### PR DESCRIPTION
**Pull request by bgiori**

### CODE REVIEW

File: docs/assets/images/experiment/evaluation-proxy.drawio.svg
Comment (nirmal91) [inline docs/assets/images/experiment/evaluation-proxy.drawio.svg:1]: should the direction of data from `experiment` and `cohorts API` be the other way around? pointing to Evaluation Proxy?

File: docs/experiment/infra/evaluation-proxy.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+## Configuration
+
+The evaluation proxy requires keys as environment variables to run. Otherwise, the service crashes on startup.
+
+| Environment Variable | Description |
+| --- | --- |
+| `AMPLITUDE_API_KEY` | The project's [API key](../../guides/amplitude-keys-guide.md#api-key). |
+| `AMPLITUDE_SECRET_KEY` | The project's [secret key](../../guides/amplitude-keys-guide.md#secret-key). |
+| `AMPLITUDE_DEPLOYMENT_KEY` | <span style="max-width:450px;display:inline-block">The key for the deployment to manage. The [deployment key](../../guides/amplitude-keys-guide.md#deployment-key) must exist within the same project as the API and secret key.</span> |
```
Comment (nirmal91) [inline docs/experiment/infra/evaluation-proxy.md:28]: maybe, call it AMPLITUDE_EXPERIMENT_DEPLOYMENT_KEY to make it more distinguishable here? I followed the link to the keys-guide page and it looks fine there

**Review by nirmal91 — APPROVED** [review]

File: docs/assets/images/experiment/evaluation-proxy.drawio.svg
Comment (bgiori) [reply docs/assets/images/experiment/evaluation-proxy.drawio.svg:1]: My logic for arrow directions is from client --> server. Regardless of HTTP request method/data flow.

File: docs/experiment/infra/evaluation-proxy.md (hunk 1 above)
Comment (justin-fiedler) [reply docs/experiment/infra/evaluation-proxy.md:28]: +1

**Review by justin-fiedler — APPROVED** [review]

**Pull request merged**